siliconcompiler/tools/openroad/scripts/sc_apr.tcl

```diff
@@ -240,7 +241,7 @@ set openroad_ifp_tie_separation [lindex [dict get $sc_cfg tool $sc_tool task $sc
 set openroad_pdn_enable [lindex [dict get $sc_cfg tool $sc_tool task $sc_task {var} pdn_enable] 0]
 
 set openroad_psm_enable [lindex [dict get $sc_cfg tool $sc_tool task $sc_task {var} psm_enable] 0]
-set openroad_psm_skip_nets [lindex [dict get $sc_cfg tool $sc_tool task $sc_task {var} psm_skip_nets] 0]
```

Comment:
What is this change for?

Reply:
Yea makes sense. I was just confused as to why before that you were getting only the first element...